bindings/python/pymongocrypt/version.py

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = '1.1.0.dev0'
+__version__ = '1.1.0b0'
 
-_MIN_LIBMONGOCRYPT_VERSION = '1.1.0beta1'
+MIN_LIBMONGOCRYPT_VERSION = '1.1.0-beta1'
```

Comment:
The old version worked for the version comparison since the parser essentially ignores the hyphens within a patch version but this still needed to be fixed to give the user a convenient way to get the libmongocrypt tag.